code/script.js

```diff
@@ -222,104 +225,130 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+// This function randomly selects a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+ secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
-// This function to start (and restart) the game
+// Function for starting and restarting the game. All charactersInPlay are shown, the secret person is set. 
+// If restarting the win or loose screen will be hidden
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
+  selectQuestion()
+  winOrLose.classList.remove("show-win-or-lose");
 }
```

Comment:
Here you can add `board.style.display = 'flex'` in order to display the board again if you hide it in the checkMyGuess function.